app/services/cloud_watch_service.rb

```diff
@@ -27,6 +27,9 @@ def past_week_metrics_data
     nil
   end
 
+  # The code to look up metrics using the old namespace can be removed from
+  # July 1st 2026 as these metrics will no longer exist in CloudWatch
```

Comment:
Possibly this comment makes most sense inside the combined_daily_starts and combined_daily_submissions methods so it's clearer what it's referring to?